README.md

```diff
@@ -18,6 +18,7 @@ Note: The full image contains only Contiv components. Installing Docker Swarm wi
 * Change directories to the extracted folder <br>`cd contiv-$VERSION`
 * To install Contiv with Docker Swarm:<br> `./install/ansible/install_swarm.sh -f cfg.yml -e <ssh key> -u <username> -i`
```

Comment:
Can you also update these two as well to say this docker swarm install is only for legacy swarm mode, and is incompatible with native swarm and v2 plugin?